cips/cip-19.md

```diff
@@ -311,6 +311,49 @@ Namespace data may span over multiple rows, in which case all the data is encaps
 containers. This enables parallelization of namespace data retrieval and certain [compositions](#protocol-compositions)
 may get advantage of that by requesting containers of a single namespace from multiple servers simultaneously.
 
+#### NamespaceRangeID
```

Comment:
namespace range may suggest it's a range of namespaces, like 0x00000001 up to 0x00000004. Maybe just NamespaceData would be better? just dropping Row prefix from RowNamespaceData, also better showing correlation between the two?

Reply:
We have:
`RowNamespaceData` - namespace data within a row
`NamespaceData` - all the namespace data that can span over multiple rows

To avoid namespace confusion I propose to do:
`NamespaceDataRange` - namespace data range that can span over multiple rows

Reply:
Agree with @Wondertan. The container itself and request namings should contain `Range`